NEWS

```diff
@@ -1,3 +1,210 @@
+5.0.0 Release Notes
+-------------------
+
+ UPDATES
+ -------
+
+ o Introduced new API in proj.h.
+   - The new API is orthogonal to the existing proj_api.h API and the
+     internally used projects.h API.
+   - The new API adds the ability to transform spatiotemporal
+     coordinates.
+   - Functions in the new API uses the "proj_" namespace.
+   - Datatypes in the new API uses the "PJ_" namespace, with a few
+     historic exceptions such as XY, XYZ, LP and LPZ.
+
+ o Introduced the concept of "transformation pipelines" that makes it
+   possible to do complex geodetic transformations of spatiotemporal
+   coordinates in PROJ by daisy chaining simple coordinate
+   operations.
+
+ o Introduced cct, the Coordinate Conversion and Transformation
+   application.
+
+ o Introduced gie, the Geospatial Integrity Investigation Environment.
+   - Selftest invoked by -C flag in proj has been removed
+
+ o Adopted terminology from the OGC/ISO-19100 geospatial standards
+   series. Key definitions are:
+   - A coordinate operation is a change of coordinates, based on a
+     one-to-one relationship, from one coordinate reference system
+     to another
+   - A transformation is a coordinate operation in which the two
+     coordinate reference systems are based on different datums, e.g.
+     a change from a global reference frame to a regional frame.
+   - A coordinate operation is an operation in which both coordinate
```

Comment:
a bit confusing: coordinate operation is defined in first and third items

Reply:
Oh yes, that is confusing. Must have slipped when I edited it. Good catch. I'll keep the top one.